system/lib/wasmfs/wasmfs.h

```diff
@@ -43,6 +46,23 @@ class WasmFS {
 
   // Returns root directory defined on WasmFS singleton.
   std::shared_ptr<Directory> getRootDirectory() { return rootDirectory; };
+
+  // Handle represents an RAII wrapper object. Access to the cwd requires
+  // acquiring a locked Handle. A Handle holds the global state's lock for the
```

Comment:
Why does this mention `cwd` in particular? (It would be good to clarify why a lock is needed for that, but not for the other child of this class, the root dir.)

Reply:
Oh I see, yeah perhaps that is sufficient.